.github/workflows/repository-cleanup.yml

```diff
@@ -0,0 +1,28 @@
+# **what?**
+# Cleanup branches left over from automation and testing.  Also cleanup
+# draft releases from release testing.
+
+# **why?**
+# The automations are leaving behind branches and releases that clutter
+# the repository.  Sometimes we need them to debug processes so we don't
+# want them immediately deleted.
+
+# **when?**
+# Mainly on a schedule of 12:00 Saturday.
+# Manual trigger can also run on demand
+
+name: Repository Cleanup
+
+on:
+  schedule:
+    - cron: '0 12 * * SAT' # At 12:00 on Saturday.
```

Comment:
let's add a little more to this comment regarding the choice of time and any potential risks associated with running this job during an active release.

Reply:
I had that in the `why` above.